region/brinstar/green/Green Hill Zone.json

```diff
@@ -55,6 +55,11 @@
       "id": "A",
       "name": "G-Mode Morph - Remote Acquire the item",
       "obstacleType": "abstract"
+    },
+    {
+      "id": "B",
```

Comment:
At some point it might be worth considering swapping the letters for obstacles "A" and "B", so that "A" could be used for the more basic one that is visible on the room diagram. Not necessary in this PR though

Reply:
I was on the fence on if this should be done. There are also some nodes in rooms that were deleted, and the higher numbers could be lowered. But that has more problems with the tools that use that data (like map tiles)